src/lib/tls/tls13_pqc/hybrid_public_key.cpp

```diff
@@ -30,15 +30,33 @@ std::vector<std::pair<std::string, std::string>> algorithm_specs_for_group(Group
          return {{"Curve25519", "Curve25519"}, {"Kyber", "Kyber-512-r3"}};
       case Group_Params::HYBRID_X25519_KYBER_768_R3_OQS:
          return {{"Curve25519", "Curve25519"}, {"Kyber", "Kyber-768-r3"}};
+      case Group_Params::HYBRID_X25519_eFRODOKEM_640_SHAKE_OQS:
+         return {{"Curve25519", "Curve25519"}, {"FrodoKEM", "eFrodoKEM-640-SHAKE"}};
+      case Group_Params::HYBRID_X25519_eFRODOKEM_640_AES_OQS:
+         return {{"Curve25519", "Curve25519"}, {"FrodoKEM", "eFrodoKEM-640-AES"}};
```

Comment:
They're taking the hobbits to TLS 1.3. Nice!

Reply:
On that note: We should rebase this to `master` and add FrodoKEM to the nightly online interoperability test against the oqs implementation.